pkg/sdk/testint/resource_monitors_integration_test.go

```diff
@@ -256,11 +280,11 @@ func TestInt_ResourceMonitorAlter(t *testing.T) {
 		alterOptions := &sdk.AlterResourceMonitorOptions{
 			Set: &sdk.ResourceMonitorSet{
 				CreditQuota: &creditQuota,
+				NotifyUsers: &sdk.NotifyUsers{
```

Comment:
https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/2316 was about "Triggers" and according to the issue desc they should be moved as you did with "NotifyUsers".